src/rds.py

```diff
@@ -0,0 +1,38 @@
+import re
+import urllib.request
+from bs4 import BeautifulSoup
+from datetime import datetime
+import json
+
+dbs = {
+    "mysql": "https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html",
+    "postgresql": "https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-release-calendar.html",
+}
+
+
+def parse_date(d):
+    return datetime.strptime(d, "%d %B %Y").strftime("%Y-%m-%d")
+
+
+for db, url in dbs.items():
+    print(f"::group::{db}")
+    releases = {}
+
+    with urllib.request.urlopen(url, data=None, timeout=5) as contents:
+        html = contents.read().decode("utf-8")
+        soup = BeautifulSoup(html, features="html5lib")
+
+        for table in soup.find_all("table"):
+            for row in table.find_all("tr"):
+                columns = row.find_all("td")
+                if len(columns) == 4:  # Only match the 'Supported XXX minor versions' table
```

Comment:
Mmmh, think I should not have changed this condition. In order to have the exact release date we must also match the "Supported xxx major versions" table. Will fix it.

Reply:
Fixed.